### PR TITLE
Feature/kon 404 provenu duplicates

### PR DESCRIPTION
**Pull request by lats1**

### CODE REVIEW

File: config/packages/security.yaml [hunk 1]
```diff
@@ -29,3 +29,7 @@ security:
         - { path: ^/saml, role: IS_AUTHENTICATED_ANONYMOUSLY }
         - { path: ^/bi/, role: ROLE_BI }
         - { path: ^/, role: ROLE_USER }
+
+    access_decision_manager:
+        strategy: unanimous
+        allow_if_all_abstain: false
```
Comment (rimi-itk) [inline config/packages/security.yaml:35]: This configuration does not seem related to anything else in this pull request. Is it added by mistake?

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: config/packages/security.yaml (hunk 1 above)
Comment (lats1) [reply config/packages/security.yaml:35]: I introduce a new voter in the core bundle that is used alongside another voter. So to make sure that every voter counts, this setting is made.

Comment (lats1) [reply config/packages/security.yaml:35]: https://symfony.com/doc/current/security/voters.html#changing-the-access-decision-strategy

Comment (rimi-itk) [reply config/packages/security.yaml:35]: Ah, there is a related PR (https://github.com/aakb/kontrolgruppen-core-bundle/pull/232)!

**Review by rimi-itk — APPROVED** [review]
